packages/config/src/layer2s/loopring.ts

```diff
@@ -256,6 +256,12 @@ export const loopring: Layer2 = {
       description:
         'The rollup owner can submit blocks, set rollup parameters and shutdown the exchange.',
     },
+    {
+      name: 'LoopringV3',
+      accounts: [discovery.getPermissionedAccount('ExchangeV3', 'loopringV3')],
+      description:
+        'Contract managing LRC staking for exchanges (one Loopring contract can manage many exchanges). Its owner (ProxyOwner) can use this contract to set the forced withdrawal fee and change the Verifier address.',
```

Comment:
Sorry, I think my comment wasn't clear. I meant the second sentence `Its owner (ProxyOwner) can use this contract to set the forced withdrawal fee and change the Verifier address.` should be a part of ProxyOwner description in the permissions section. LoopringV3 should stay in contracts section IMO

Reply:
ohh ok will do